orchestra/tests/helpers/fixtures.py

```diff
@@ -442,32 +495,11 @@ def setup_task_history(test):
     second_assignment.snapshots['snapshots'] = (
         deepcopy(third_assignment.snapshots['snapshots']) +
         second_assignment.snapshots['snapshots'])[:-1]
-
-    def fix_datetimes(snapshots, new_datetimes):
```

Comment:
Removing this, since it was only used in the analytics test (which needs to be completely rewritten)